lightning/src/routing/scoring.rs

```diff
@@ -331,6 +331,17 @@ pub struct ProbabilisticScoringParameters {
 	/// Default value: 500 msat
 	pub base_penalty_msat: u64,
 
+	/// A fixed penalty in msats to apply to each channel, multiplied by the payment amount.
+	///
+	/// The purpose of the amount penalty is to avoid having fees dominate the channel cost (i.e.,
+	/// fees plus penalty) for large payments. The penalty is computed as the product of this
+	/// multiplier and `2^30`ths of the payment amount.
+	///
+	/// ie `amount_penalty_multiplier_msat * amount_msat / 2^30`
+	///
+	/// Default value: 8,192 msat
+	pub base_penalty_amount_multiplier_msat: u64,
```

Comment:
We should note that this is added to `base_penalty_msat` as it might surprising that these aren't mutually exclusive. But maybe we should make them so? Workaround is to set either to zero, of course.